src/test/regress/expected/non_super_user_object_metadata.out

```diff
@@ -0,0 +1,455 @@
+SET citus.enable_ddl_propagation TO OFF;
+CREATE SCHEMA local_schema;
+SET search_path TO local_schema;
+SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
```

Comment:
Ozan just merged a PR that enables MX for majority of the tests in multi_1_schedule. 

If we add your test to `turn_mx_on` section (see the multi_1_schedule), we shouldn't need start/stop metadata sync calls.

Would be nice to have this test in MX enabled section.